sdk/tables/Azure.Data.Tables/tests/TableClientTests.cs

```diff
@@ -102,13 +98,5 @@ public void GetSasBuilderPopulatesRawPermissionsAndExpiry()
             Assert.That(sas.Permissions, Is.EqualTo(permissions.ToPermissionsString()));
             Assert.That(sas.ExpiresOn, Is.EqualTo(expiry));
         }
-
-        public class MinEntity : ITableEntity
```

Comment:
Might still be worth testing against this case since it'll run through the strongly typed path?

Reply:
Since the tests are checking the parameters before any distinction between dictionary and strongly-typed, would there be any difference? The way that the entity properties are being checked is through the `ITableEntity` accessors, so I wasn't sure if it would matter.

Reply:
You're right - we're pulling those from the interface so there's no difference.